lib/components/carousel.vue

```diff
@@ -139,27 +140,35 @@
             },
             value: {
                 type: Number,
-                defrault: 0
```

Comment:
This was the main culprit of carousel not showing initially, as value defaulted to `null`/`undefined` if a v-model wasn't bound.